lib/cmd/clean.cc

```diff
@@ -0,0 +1,48 @@
+// std
+#include <iostream>
+#include <string>
```

Comment:
I don't think these includes & `poac/data/manifest.hpp` are needed here.

Reply:
It had copied and pasted then forgotten to erase... I've removed it

Reply:
Never mind. Thank you!